src/generic_core/social.ts

```diff
@@ -626,6 +626,32 @@ export function notifyUI(networkName :string, userId :string) {
       });
     }
 
+    public processCloudNetworkData(networkData :any) :string {
```

Comment:
I think a name like `augmentCloudInviteWithChromePolicy` would be more descriptive (it's long but this is a very weird, special function).